src/main/java/teammates/common/datatransfer/attributes/FeedbackSessionAttributes.java

```diff
@@ -52,6 +53,8 @@
         respondingInstructorList = new HashSet<>();
         respondingStudentList = new HashSet<>();
 
+        timeZone = ZoneId.of("UTC");
```

Comment:
Should this be extracted to `Const` as something like `DEFAULT_TIME_ZONE`? A similar fall back is used in `CoursesDb`.